code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -375,6 +375,16 @@
 	cost = 25 /// A fresh start, but a start with nothing. Hard to use as well
 	species = list("Grey")
 
+//Unathi
+/datum/uplink_item/species_restricted/prescan
+	name = "Breach Cleaver"
+	desc = "This massive blade harkons back to the wars on Moghes. Wielding it imbues you with the unquenchable desire for martial prowess. \
+	Two hands, comes in a scabbard. Has different effects based on intent."
```

Comment:
```suggestion
	Requires two hands to whiled. Comes in a scabbard. Has different effects based on intent."
```

Reply:
> 

Should be "wield" instead of "whiled" I believe.

Reply:
indeed it should